core/src/code.rs

```diff
@@ -124,14 +124,19 @@ fn check_code(module: &Module, config: &TryNewCodeConfig) -> Result<(), CodeErro
     }
 
     if config.check_imports {
-        let syscalls = SyscallName::instrumentable_map();
+        let syscalls = SyscallName::all_map();
```

Comment:
plz revert commit 947bd0920ad8a1cfa1a5d35e5a54231d61af3bb7
I think someone forgot to add `SystemReserveGas` and `EnvVars` to `instrumentable()` and that's why tests fails

Reply:
ah ok